packages/veritone-react-common/src/components/FolderTree/Searchbox/index.js

```diff
@@ -30,23 +30,44 @@ export default function Searchbox({
     }
   }
   function handleChangeInput(event) {
+    if (event.target.value !== '') {
+      setShowCloseIcon(true);
+    } else {
+      setShowCloseIcon(false);
+    }
     setData(event.target.value);
   }
   function handleKeyDown(e) {
     if (e.key === 'Enter') {
       onSearch(data);
     }
   }
+  function handleRemoveText(e) {
+    setData('');
+    setShowCloseIcon(false);
+    onSearch('');
+  }
   return (
     <div className={classes.searchBoxRoot}>
       <InputBase
         onChange={handleChangeInput}
         inputRef={inputRef}
+        value={data}
         className={classes.input}
         placeholder={placeholder}
         inputProps={{ 'aria-label': 'Search-input' }}
         onKeyDown={handleKeyDown}
       />
+      {showCloseIcon ? (
```

Comment:
Can't we just do `!!data` instead of presenting a new showCloseIcon variable?

Reply:
done.